ManipulationHelpers/ManipulationHelpers.d.ts

```diff
@@ -0,0 +1,276 @@
+/// <reference path="../typings/babylon.d.ts" />
+declare module ManipulationHelpers {
+    import Color3 = BABYLON.Color3;
+    import Scene = BABYLON.Scene;
+    import Vector3 = BABYLON.Vector3;
+    import Quaternion = BABYLON.Quaternion;
+    import Vector2 = BABYLON.Vector2;
+    const enum RadixFeatures {
+        None = 0,
+        /**
+         * Display the Arrow that follows the X Axis
+         */
+        ArrowX = 1,
+        /**
+         * Display the Arrow that follows the Y Axis
+         */
+        ArrowY = 2,
+        /**
+         * Display the Arrow that follows the Z Axis
+         */
+        ArrowZ = 4,
+        /**
+         * Display the Arrow that follows the XYZ Axis
+         */
+        ArrowsXYZ = 7,
+        /**
+         * Display the anchor that allow XY plane manipulation
+         */
+        PlaneSelectionXY = 16,
+        /**
+         * Display the anchor that allow XZ plane manipulation
+         */
+        PlaneSelectionXZ = 32,
+        /**
+         * Display the anchor that allow YZ plane manipulation
+         */
+        PlaneSelectionYZ = 64,
+        /**
+         * Display all the anchors that allow plane manipulation
+         */
+        AllPlanesSelection = 112,
+        /**
+         * Display the rotation cylinder that allows rotation manipulation along the X Axis
+         */
+        RotationX = 256,
+        /**
+         * Display the rotation cylinder that allows rotation manipulation along the Y Axis
+         */
+        RotationY = 512,
+        /**
+         * Display the rotation cylinder that allows rotation manipulation along the A Axis
+         */
+        RotationZ = 1024,
+        /**
+         * Display all rotation cylinders
+         */
+        Rotations = 1792,
+    }
+    /**
+     * This class create the visual geometry to display a manipulation radix in a viewport.
+     * It also implements the logic to handler intersection, hover on feature.
+     */
+    class Radix {
```

Comment:
@RaananW this is the generated definition file you're looking at right?
that's why there're multiple module declarations and no implem, correct?